packages/editor/src/editor/canvas/ComponentBlock.tsx

```diff
@@ -117,80 +124,84 @@ const Draggable = ({ config, data }: DraggableProps) => {
   });
 
   return (
-    <Popover open={isSelected && !isDragging}>
-      <PopoverAnchor asChild>
-        <div
-          onClick={e => {
-            e.stopPropagation();
-            setSelectedElement(data.cid);
-          }}
-          onDoubleClick={e => {
-            e.stopPropagation();
-            setUi(old => ({ ...old, properties: true }));
-          }}
-          onKeyDown={e => {
-            if (e.key === 'Enter') {
+    data.type !== 'Dialog' && (
```

Comment:
do your wrap here the whole return with data.type !== 'Dialog'?
If yes do you like this?
Why not do simply:
```tsx
if (data.type === 'Dialog') {
  return null;
} 
return (<Popover...
```